chart/kubeapps/Chart.yaml

```diff
@@ -33,4 +33,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.8.0-dev3
+version: 7.7.4
```

Comment:
@antgamdia Is this version correct?
[Bitnami chart version](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/Chart.yaml#L36)

Reply:
Nope, I don't think so. Since we added new params in the chart (in the kapp-ctrl plugin config), this should be a `minor` release according to the Content team. So I just bumped it to 7.8.0 in the first change, and then I continue modifying the devXX prerelease tag.
In short, I don't think we should modify the chart version here.

What we do have to change, though, is this version once our CI creates the PR in the bitnami/charts repo. I mean, in their repo, not in ours.

Reply:
Ok, got it! Thanks

Reply:
Done!